datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -284,6 +289,9 @@ export default function DefaultPreviewCard({
                         parentEntities={parentEntities}
                         parentContainersRef={contentRef}
                         areContainersTruncated={isContentTruncated}
+                        parentDatasetUrn={urn}
```

Comment:
i think it might be nice for `DefaultPreviewCard` to explicitly take in an optional `parentDatasetUrn` and pass that in from the schema field since right now passing in `parentDatasetUrn={urn}` would mean every entity would pass in their own urn as the `parentDatasetUrn` which could definitely cause confusion down the line

Reply:
As per the [last comment](https://github.com/datahub-project/datahub/pull/11585#discussion_r1815344337), now we are passing `parentDataset` object, so we don't require `parentDatasetUrn` to be passed explicitly.